src/richie/apps/courses/models/course.py

```diff
@@ -110,12 +110,15 @@ def __init__(self, priority, date_time=None):
         ):
             text = _("forever open")
             date_time = None
+
         kwargs = {
             "priority": priority,
             "datetime": date_time,
             "call_to_action": self.STATE_CALLS_TO_ACTION[priority],
             "text": text,
         }
+        if priority == CourseState.ONGOING_OPEN:
+            kwargs["datetime"] = None
```

Comment:
Instead you could edit the condition statement above, WDYT ?

```python
        if (
            priority in [CourseState.ONGOING_OPEN, CourseState.ARCHIVED_OPEN]
            and date_time.year == MAXYEAR
        ):
            text = _("forever open")
            date_time = None
        elif priority == CourseState.ONGOING_OPEN:
            date_time = None
```

Reply:
I think this is a better idea, most of the times i have tried to follow pattern of the code and same parts a saw some if's.
In this exactly point i take a time to think do or don't hahaha, sorry for that.
Thank you, for revision, tomorrow at morning i will made changes at commit of this PR.